aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -412,7 +412,7 @@ def export_tree(
     }
 
     # N.B. We're really calling zipfolder.open (if exporting a zipfile)
-    with folder.open('data.json', mode='w') as fhandle:
+    with folder.open('data.json', mode='w', encoding='utf-8') as fhandle:
```

Comment:
The main question is under what conditions did @AntimoMarrazzo case fail and why was this not caught by the tests. I would first want to fully understand the failure and then add a test.